Cargo.toml

```diff
@@ -124,6 +124,10 @@ test-llvm = [
 # context where signal handling is a problem, such as tarpaulin for code coverage.
 test-no-traps = ["wasmer-wast/test-no-traps"]
 
+[profile.release.package.wasmer-c-api]
+opt-level = "z"
```

Comment:
`z` will prioritize code size over execution speed, something to think about.  Cargo recently introduced custom release profiles, I believe they're still nightly only though.  so in the future it'll be trivial to have settings for "smallest binary" vs "fastest binary"

Reply:
I know. Do we favor speed over size?